tests/test_workers/test_pkg_managers/test_pip.py

```diff
@@ -17,6 +18,7 @@
 from tests.helper_utils import write_file_tree
 
 
+THIS_MODULE_DIR = Path(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
You could do this without `os.path`:
```suggestion
THIS_MODULE_DIR = Path(__file__).resolve().parent
```